kubernetes/test-infra/eks/main.tf

```diff
@@ -21,22 +67,26 @@ module "vpc" {
 
 module "cluster" {
   source  = "terraform-aws-modules/eks/aws"
-  version = "v13.2.1"
+  version = "14.0.0"
 
   vpc_id  = module.vpc.vpc_id
   subnets = module.vpc.subnets
 
   cluster_name    = module.vpc.cluster_name
   cluster_version = var.kubernetes_version
-  manage_aws_auth = false
-  # This kubeconfig expires in 15 minutes, so we'll use another method.
+  manage_aws_auth = false # Managed in ./kubernetes-config/main.tf instead.
+  # This kubeconfig expires in 15 minutes, so we'll use an exec block instead.
+  # See ./kubernetes-config/main.tf provider block for details.
   write_kubeconfig = false
 
+  workers_group_defaults = {
+    root_volume_type = "gp2"
```

Comment:
GP2 volumes are generally more expensive. 
Is there a technical need for nodes to run on GP2 volumes? We're likely not putting much workload on the nodes during our tests.

Reply:
gp2 was the default in the EKS module, but they recently updated it to gp3. I put it back to the old default because gp3 isn't available in all regions.

Reply:
This is the commit where it was added. https://github.com/terraform-aws-modules/terraform-aws-eks/commit/76537d1b8d89dbe104ec32fb6ddd2d7c084b5e28

Reply:
Oh, if that's the case then disregard my comment. I must have based it on old information and likely GPx volumes are now the only choice. Sorry for the confusion :)